src/main/menu.ts

```diff
@@ -176,6 +176,30 @@ export default class MenuBuilder {
         },
         enabled: false, // by default, gets updated when a project is entered
       },
+      {
+        id: 'playPause',
+        label: 'Play Or Pause Video',
```

Comment:
At the very least, `Play or Pause Video`

But better would be `Play/Pause Video`

Reply:
Okay